### DISCUSSION TITLE
Fitting error

### DISCUSSION THREAD

**Discussion by chlwjd1234** [ttadano/alamode#29] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/52664564/119769160-f2383680-bef4-11eb-8dcd-3c032d9be7c6.png)

The error keep occurs, and I don't understand why rank becomes deficient . It says MKL has problem too.., then is it related to compile error? Below is compile log
![image](https://user-images.githubusercontent.com/52664564/119769694-dc774100-bef5-11eb-8073-6b3c6edc127a.png)

What I doubt is that the compiler  is detected as gnu first , but with BLAS  , MKL is used . Can this problem cause the MKL DGGLSE error?

**Comment by ttadano** [comment, marked as answer]
Perhaps, the error occurred because of the rank deficiency of the given matrix, as indicated by the warning message. Please try again with another algorithm:

- SVD (dense solver): `ICONST = 11`
- Sparse solver :`SPARSE = 1`

If you still observe a similar error with these algorithms, you will need to increase the size of the training dataset.
